validator/etc/txnvalidator.js

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
This file should not have been checked in - it's old and you should just delete your local copy.